BackEndApp/src/main/java/com/example/backendapp/Controller/PlantProfileController.java

```diff
@@ -9,31 +9,43 @@
 
 @RestController
 @RequiredArgsConstructor
-public class PlantProfileController {
+public class  PlantProfileController {

```

Comment:
Remove the extra space. To make sure you indented code in the correct way, use this [guide](https://stackoverflow.com/questions/9044472/how-to-correct-indentation-in-intellij).